manifests/r/RescueTime,Inc/RescueTime/2.16.8.2/RescueTime,Inc.RescueTime.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: RescueTime,Inc.RescueTime

```

The comma isn't necessary here
```suggestion
PackageIdentifier: RescueTimeInc.RescueTime
```